src/test/crimson/perf_crimson_msgr.cc

```diff
@@ -41,7 +40,8 @@ enum class perf_mode_t {
 
 static seastar::future<> run(unsigned rounds,
                              unsigned jobs,
-                             unsigned bs,
+                             unsigned cbs,
+                             unsigned sbs,
```

Comment:
i'd suggest  more descriptive names as i think the larger the lexical scope of a variable has, a better name it deserves.

Reply:
The options are now wrapped with struct, see https://github.com/cyx1231st/ceph/commit/b0db7d7ff912c3d5bd6d48b16fdd1144364d16d7.